cmake/posix/px4_impl_posix.cmake

```diff
@@ -196,8 +196,17 @@ function(px4_os_add_flags)
 				)
 		endif()
 
-	else()
+	elseif(CYGWIN)
+		set(added_definitions
+			-D__PX4_POSIX
```

Comment:
You can move `-D__PX4_POSIX` to the top of this file so that it's applied to all configs.

`-Dnoreturn_function=__attribute__\(\(noreturn\)\)` is also added for all posix configs.

Reply:
ok, let me see if there's an easy way.